.travis.yml

```diff
@@ -5,21 +5,20 @@ matrix:
     # Linux Python 2
     - sudo: required
       language: python
+      python: "2.7"
       services:
         - docker
       env:
-        - "PYTHON=python2"
+        - "PYTHON=python"
 
     # Linux Python 3
     - sudo: required
       language: python
+      python: "3.4"
       services:
         - docker
       env:
-        - "PYTHON=python3"
-      before_install:
-        - sudo apt-get -qq update
-        - sudo apt-get install -y python3-pip
+        - "PYTHON=python"
```

Comment:
If distinction between `python2` and `python3` is not needed, we can just get rid of this environment variable

Reply:
This is still needed for macOS which uses `python2` and `python3` (might not show in the diff)

Reply:
Sorry, should've looked better!